openassessment/templates/openassessmentblock/edit/oa_edit_header_and_validation.html

```diff
@@ -2,22 +2,51 @@
 <div id="openassessment_editor_header">
     <h6 id="oa_editor_window_title" class="title modal_window_title" >{% trans "Open Response Assessment" %}</h6>
     <ul class="editor_modes action_list action_modes editor_tabs">
-        <li class="view-button oa_editor_tab" id="oa_edit_prompt_tab"><a href="#oa_prompt_editor_wrapper">{% trans "Prompt" %}</a></li>
-        <li class="view-button oa_editor_tab" id="oa_edit_rubric_tab"><a href="#oa_rubric_editor_wrapper">{% trans "Rubric" %}</a></li>
-        <li class="view-button oa_editor_tab" id="oa_edit_schedule_tab"><a href="#oa_schedule_editor_wrapper">{% trans "Schedule" %}</a></li>
-        <li class="view-button oa_editor_tab" id="oa_edit_assessment_steps_tab"><a href="#oa_assessment_steps_editor_wrapper">{% trans "Assessment steps" %}</a></li>
-        <li class="view-button oa_editor_tab" id="oa_edit_settings_tab"><a href="#oa_settings_editor_wrapper">{% trans "Settings" %}</a></li>
+        <li role="tab" class="view-button oa_editor_tab" id="oa_edit_prompt_tab">
+            <a href="#oa_prompt_editor_wrapper">
+                <span class="validation-warning" aria-label="">{% include 'openassessmentblock/icons/warning_filled.html' %}</span>
```

Comment:
should this include `aria-label="warning"`?

Reply:
@wittjeff's requirements said to leave that as `aria-label=""`